tests/it/main.rs

```diff
@@ -0,0 +1,3 @@
+mod e2e_rpc_server;
```

Comment:
🎉 Thanks for consolidating the integration tests into a single binary. This will scale better as we add more tests, since there will be less linker runs.